modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/css/Actions.scss

```diff
@@ -1,7 +1,7 @@
 @import 'atlas-variables';
 @import 'cadmin-variables';
 
-html#{$cadmin-selector} {
+html#{$cadmin-selector} .cadmin {
```

Comment:
This `.cadmin` part was removed recently and was breaking the Actions UI

Reply:
Will this cause a regression on [LPD-1376](https://liferay.atlassian.net/browse/LPD-1376)? @kresimir-coko said in https://github.com/liferay/liferay-portal/commit/388ef3ebbedc615ee0e8e301cefa47b9dfb4211e commit message: 
> remove .cadmin selector because the modal is outside the element that has the cadmin class

Reply:
@juanjofgliferay  Reading on https://github.com/liferay-frontend/liferay-portal/pull/4577#issuecomment-2491339566, I don't understand why are there two cases? Don't we have just one icon selection in DSM?

Reply:
If `panel-unstyled` affects something other then icon selection, then yeah, something like two cases you are suggesting sounds good.

Reply:
>  I don't understand why are there two cases. Don't we have just one icon selection in DSM?

`panel-unstyled` affects every section of the Actions form. It is not about the icon selection modal.

With the our markup, the `html#{$cadmin-selector} .cadmin` will not apply styles to the modal as it is not a `.cadmin` descendant.

```html
<div id="main">
  <div class="cadmin fds-view">
    <div class="panel panel-unstyled">
      [form elements]
    </div>
  </div>
</div>
...
<div class="dsm-actions-icon-selection-modal">
  [icons]
</div>
```
This is why we may want a couple of cases.